docs/conf.py

```diff
@@ -295,3 +295,12 @@
 # ------------------------------------
 
 hoverxref_auto_ref = True
+hoverxref_project = "scrapy"
+hoverxref_version = release
+hoverxref_role_types = {
```

Comment:
This seems to be necessary too, otherwise we get

```
Using default style for unknown typ. Define it in hoverxref_role_types. typ=ref style=tooltip
```
Setting: 
https://sphinx-hoverxref.readthedocs.io/en/latest/configuration.html#confval-hoverxref_role_types
Failed build: https://travis-ci.org/github/scrapy/scrapy/jobs/673579524#L1209